globalSetup.js

```diff
@@ -0,0 +1,4 @@
+export default async () => {
+    // eslint-disable-next-line no-undef
```

Comment:
has to disable the 'process in not defined' error since it's defined for `jest` despite it's not defined for `vite`. The code's working properly.